lib/services/ecs/ecs-service-template.yml

```diff
@@ -161,6 +161,13 @@ Resources:
       EvaluationPeriods: '1'
       Namespace: Handel/ECS
       Threshold: '0'
+  EcsAutoScalingLifecycle:
+    Type: AWS::AutoScaling::LifecycleHook
```

Comment:
I've not used these lifecycle hooks myself before. How does this invoke the draining Lambda you've defined?

Reply:
It doesn't. It just causes the termination process to pause until it either receives a continue or times out. Currently I'm just letting it timeout. When the autoscaling group sets an ec2 instance to terminate the draining lambda gets invoked marking putting the ec2 instance into the draining status.